README.md

```diff
@@ -1,13 +1,14 @@
+<img src="https://sajjaadf.github.io/chromebrew-Dev/brew.png" alt="CBlogo" width="123.5" height="145.75">
```

Comment:
A better way to do this is to upload an already resized image to `/images/brew.png` and reference that instead of adjusting the height and width.  Generally this is considered bad practice because the image is not optimized.

Reply:
Okay